tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -362,10 +362,19 @@ def _FieldTranslationIsValid(self, qualified_field_name: str,
         return False
 
       is_valid = True
-      for state in ft.states.keys():
+      raw_values_found = set()
+      for state, value in ft.states.items():
         if state not in valid_states:
           print(f'Field {qualified_field_name} has an invalid state: {state}')
           is_valid = False
+        raw_values = value if isinstance(value, list) else [value]
+        for raw_value in raw_values:
+          if raw_value in raw_values_found:
+            print(f'Raw value {raw_value} is assigned to multiple states for '
```

Comment:
is this an error message?

Reply:
@charbull  It is an error.  If they defined OPEN = 1 and CLOSED = 1, the system wouldn't know what to do when the telemetry provides a present value of 1 for that field.

Reply:
I see, can we make more precise? 
add `duplicate mapping` in the message?

Reply:
@dplute

Reply:
@charbull I have adjusted the message.  I will defer to you if you have better wording in mind.